README.md

```diff
@@ -60,28 +60,37 @@ The extension allows developers to find and fix security vulnerabilities in thei
+#### 📦 Software Composition Analysis (SCA)
+- Scan project dependencies for security issues 
+- Access enriched CVE data provided by the JFrog Security Research team.
 
-#### Advanced Scans
+Learn more about enriched CVEs in the [JFrog Security CVE Research and Enrichment](https://www.jfrog.com/confluence/display/JFROG/JFrog+Security+CVE+Research+and+Enrichment) documentation.
+
+#### 🏗️ Infrastructure as Code (IaC) Scanning 
+- Analyze Infrastructure as Code (IaC) files, such as Terraform, to identify security vulnerabilities and misconfigurations before deploying your cloud infrastructure.
+- Get actionable insights and recommendations for securing your IaC configurations.
+
+#### 🔐 Secrets Scanning
+Detect and prevent the inclusion of sensitive information, such as credentials and API keys, in your codebase.
+
+#### 🔍 Advanced Scans
 *Requires Xray version 3.66.5 or above and Enterprise X / Enterprise+ subscription with Advanced DevSecOps.*
 
 With advanced [**Contextual Analysis**](#contextual-analysis), understand the applicability of CVEs in your application and utilize JFrog Security scanners to analyze the way you use 3rd party packages in your projects.
 Automatically validate some high-impact vulnerabilities, such as vulnerabilities that have prerequisites for exploitations, and reduce false positives and vulnerability noise with smart CVE analysis.
 
 To learn more, see [here](https://www.jfrog.com/confluence/display/JFROG/Vulnerability+Contextual+Analysis).
 
-#### Supported Packages
+#### 🛡️ Supported Packages
 | Features                                             | [Go](#go-projects) | [Maven](#maven-projects) | [npm](#npm-projects) | [Yarn v1](#yarn-v1-projects) | [Pypi](#pypi-projects) | [.NET](#net-projects) |
 |---------------------------------------------------|:----:|:------:|:-------:|:----:|:--------:|:-------:|
-| SCA                                               |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
-| Upgrade vulnerable dependencies to fixed versions |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
-| Contextual Analysis                               |  ❌  |   ❌   |   ✅    |  ✅  |    ✅    |   ❌    |
-| Exclude dev dependencies                          |  ❌  |   ❌   |   ✅    |  ❌  |    ❌    |   ❌    |
+| [SCA](#-software-composition-analysis-sca)                                               |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
+| [Upgrade vulnerable dependencies to fixed versions](#updating-dependencies) |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
+| [Contextual Analysis](#-advanced-scans)                               |  ❌  |    ✅   |   ✅    |  ✅  |    ✅    |   ❌    |
+| [Secrets Scanning](#-secrets-scanning)                          |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
+| [Exclude dev dependencies](#exclude-development-dependencies-during-scan)                          |  ❌  |   ❌   |   ✅    |  ❌  |    ❌    |   ❌    |
```

Comment:
please fix links in this table, it lead to page 404
Are you sure secrets scanning support all types of packages? are you sure its even related to a packgetype? maybe we should validate this first

Reply:
Yes, secrets scanning supports all types of packages